astropy/wcs/docstrings.py

```diff
@@ -320,6 +340,290 @@ def RA_DEC_ORDER(indent=0):
+*Read-only.* True if the projection is equal area.
+"""
+
+prjprm_conformal = """
+*Read-only.* True if the projection is conformal.
+"""
+
+prjprm_global_projection = """
+*Read-only.* True if the projection can represent the whole sphere in a finite,
+non-overlapped mapping.
+"""
+
+prjprm_divergent = """
+*Read-only.* True if the projection diverges in latitude.
+"""
+
+prjprm_x0 = r"""
+*Read-only.* The offset in ``x`` used to force :math:`(x,y) = (0,0)` at
+:math:`(\phi_0, \theta_0)`.
+"""
+
+prjprm_y0 = r"""
+*Read-only.* The offset in ``y`` used to force :math:`(x,y) = (0,0)` at
+:math:`(\phi_0, \theta_0)`.
+"""
+
+prjprm_m = """
+*Read-only.* Intermediate integer value (used only for the ``ZPN`` and ``HPX`` projections).
+"""
+
+prjprm_n = """
+*Read-only.* Intermediate integer value (used only for the ``ZPN`` and ``HPX`` projections).
+"""
+
+prjprm_set = """
+This method sets up a ``Prjprm`` object according to information supplied
+within it.
+
+Note that this routine need not be called directly; it will be invoked by
+`prjx2s` and `prjs2x` if ``Prjprm.flag`` is anything other than a predefined
+magic value.
+
+The one important property of ``set()`` is that the projection code must be
+defined in the ``Prjprm`` in order for ``set()`` to identify the required
+projection.
+
+Raises
+------
+MemoryError
+    Null ``prjprm`` pointer passed to WCSLIB routines.
+
+InvalidPrjParametersError
+    Invalid projection parameters.
+
+InvalidCoordinateError
+    One or more of the ``(x,y)`` or ``(lon,lat)`` coordinates were invalid.
+"""
+
+prjprm_prjx2s = r"""
+Deproject Cartesian ``(x,y)`` coordinates in the plane of projection to native
+spherical coordinates :math:`(\phi,\theta)`.
+
+The projection is that specified by ``Prjprm.code``.
+
+Parameters
+----------
+x : numpy.ndarray
+    Array corresponding to the first (``x``) projected coordinate.
+y : numpy.ndarray
+    Array corresponding to the second (``y``) projected coordinate.
+
+Returns
+-------
+phi,theta : tuple of numpy.ndarray
+    Longitude and latitude :math:`(\phi,\theta)` of the projected point in
+    native spherical coordinates (in degrees).
+
+Raises
+------
+MemoryError
+    Null ``prjprm`` pointer passed to WCSLIB routines.
+
+InvalidPrjParametersError
+    Invalid projection parameters.
+
+InvalidCoordinateError
+    One or more of the ``(x,y)`` coordinates were invalid.
+
+"""
+
+prjprm_prjs2x = r"""
+Project native spherical coordinates :math:`(\phi,\theta)` to Cartesian
+``(x,y)`` coordinates in the plane of projection.
+
+The projection is that specified by ``Prjprm.code``.
+
+Parameters
+----------
+phi : numpy.ndarray
+    Array corresponding to the longitude :math:`\phi` of the projected point
+    in native spherical coordinates (in degrees).
+theta : numpy.ndarray
+    Array corresponding to the longitude :math:`\theta` of the projected point
+    in native spherical coordinatess (in degrees).
+
+Returns
+-------
+x,y : tuple of numpy.ndarray
```

Comment:
https://numpydoc.readthedocs.io/en/latest/format.html#parameters  wants a space between parameters in a list.

```suggestion
x, y : tuple of ndarray
```